primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js

```diff
@@ -1344,6 +1344,7 @@
             hourText: 'Hour',
             minuteText: 'Minute',
             secondText: 'Second',
+            millisecondText: 'Millisecond',
```

Comment:
We should add this to the locale files as well. @melloware as a separate PR?

Reply:
I am fine with separate PR or if @wal-jan wants to add them to the local files in this PR.

Reply:
@jepsar , @melloware: I'll all localized texts to PR (Google translate hopefully will do the translations)

Reply:
Nice yeah at least that will be a good start and people can always complain and we can fix if its not perfect. Much appreciated.